### ISSUE TITLE
Unable to assign alert status to incident

### ISSUE THREAD

**Issue by lznger88** [simcen/alert_manager#244]
When editing an incident, under the Incident Workflow, the 'Status:' dropdown is greyed out. The user's populated under Settings>Users Settings are showing as type 'builtin' and the active user directory is set to 'both'. All users have the roles alert_manager, alert_manager_user.
I reviewed all lookups (using the Lookup editor app), ensure all users have the alert manager roles as well as following a similiar issue, though this wasnt a migration it was a fresh install on a new Splunk instance - https://github.com/simcen/alert_manager/issues/211

Splunk: 7.2.6, Alert Manager: 2.2.2, Alert Manager Add-On: 2.2.0

![image](https://user-images.githubusercontent.com/39687683/60152486-087a7b00-9824-11e9-874c-e8d707ce0a5e.png)

**Comment by HoorayJorge** [comment]
I'm having the same issue on Splunk 8.03, TA 2.2.1 and App 2.3

**Comment by my2ndhead** [comment]
Can you check if the alert_status lookup has entries?
E.g. run following search within the alert_manager app 

`| inputlookup alert_status` 

I've seen issues when the migration script hasn't correctly run and it hasn't populated the lookup. You can re-enable the script in local/inputs.conf.

For the upcoming release 3.0.0 (Python 3.7) I have fixed a few issues regarding alert status.

**Comment by HoorayJorge** [comment]
Alert_status.csv appears populated

I'm also having an issue where incident_results isn't populating regularly.  Perhaps these issues are related?

**Comment by my2ndhead** [comment]
It's not alert_status.csv that has to be populated.

Did you run the inputlookup command? What does it five back?

Can you open the Browser Developer Tools Console to check if you can see if it throws some errors?

incident_results is not related to this problem.

**Comment by HoorayJorge** [comment]
You're right alert_status didn't build correctly (there were input type errors in the "hidden" column.  Thanks for the support.

The incident_results issue was due to '.' characters in the field names.
